juntagrico/util/lifecycle.py

```diff
@@ -0,0 +1,5 @@
+def handle_activated_deactivated(instance, sender, activated, deactivated):
+    if instance._old['active'] != instance.active and instance._old['active'] is False and instance.deactivation_date is None:
```

Comment:
Probably `instance.activation_date` should be check here instead of `deactivation_date`?
However this is checked again in the next step. It should be removed here.

Reply:
no deactivation date has to be checked in order to prevent reactivation
step 0
instance.active=False  && instance.deactivationdate = none
step1 (activate)
instance.active=True && instance.deactivationdate = none
step2 (deactivate)
instance.active=False && instance deactivationdate=some
step3 (reactivation)
instance.active=True && instance.deactivationdate=some